sql/core/src/test/scala/org/apache/spark/sql/UserDefinedTypeSuite.scala

```diff
@@ -17,15 +17,22 @@
 
 package org.apache.spark.sql
 
-import java.util.Arrays
+import java.nio.file.Files
+import java.sql.Timestamp
+import java.util.{Arrays, GregorianCalendar}
 
+import com.sun.org.apache.xerces.internal.jaxp.datatype.XMLGregorianCalendarImpl
+import javax.xml.datatype.XMLGregorianCalendar
```

Comment:
Could you run `dev/scalastyle` and fix the errors?
```
[error] /home/runner/work/spark/spark/sql/core/src/test/scala/org/apache/spark/sql/UserDefinedTypeSuite.scala:26:0: There should at least one a single empty line separating groups 3rdParty and spark.
15
[error] /home/runner/work/spark/spark/sql/core/src/test/scala/org/apache/spark/sql/UserDefinedTypeSuite.scala:35:0: scala.reflect.ClassTag should be in group scala, not spark.
```

Reply:
Thank you for the pointers. I've fixed the style errors.